templates/db.inifile.params.j2

```diff
@@ -181,7 +181,7 @@ NW_readProfileDir.profileDir = {{ sapmnt_path }}/{{ sid.upper() }}/profile/
 archives.downloadBasket = {{ download_basket }}
 
 # Specify whether a Landscape Reorg Check Procedures file 'HdbLandscapeReorgCheckProcedure.SQL' is to be used. Possible values are 'USEFILE', 'DONOTUSEFILE','DONOTRUN'. Default value is 'USEFILE'.
-# hanadb.landscape.reorg.useCheckProcedureFile = USEFILE
+hanadb.landscape.reorg.useCheckProcedureFile = DONOTUSEFILE
```

Comment:
How did you find out this?
Is there any guide that you followed?
Just to make sure that we don't create any other inconvenience with this change

Reply:
When not setting this, there was the error message that `HdbLandscapeReorgCheckProcedure.SQL` could not be found.
Looking at the comments of the ini file it was quite obvious what to change to fix it.
There is also a similar line already existing: https://github.com/SUSE/sapnwbootstrap-formula/blob/cf7d0620a6841059b423231d2c4aad7646a9e480/templates/db.inifile.params.j2#L187

Sadly I could not fine any documentation about this parameter.

Reply:
Ok, I get it.
I was just wondering if we could put the error message somewhere, to make it even more explicit what this change is fixing.
In any case, let's move forward with the solution

Reply:
Added an additional comment right above the parameter.